Testscripts/Windows/VERIFY-XDP-ACTION.ps1

```diff
@@ -47,21 +45,13 @@ function Main {
         Write-LogInfo "  Internal IP : $($senderVMData.InternalIP)"
 
         # Check for compatible kernel
-        $currentKernelVersion = Run-LinuxCmd -ip $receiverVMData.PublicIP -port $receiverVMData.SSHPort `
-                -username $user -password $password -command "uname -r"
-        # ToDo: Update Minimum kernel version check once patches are in downstream distro.
-        if ($global:DetectedDistro -eq "UBUNTU"){
-            if ((Compare-KernelVersion $currentKernelVersion $MIN_KERNEL_VERSION) -lt 0){
-                Write-LogInfo "Unsupported kernel version: $currentKernelVersion"
-                return $global:ResultSkipped
-            }
-        } elseif ($global:DetectedDistro -eq "REDHAT"){
-            if ((Compare-KernelVersion $currentKernelVersion $RHEL_MIN_KERNEL_VERSION) -lt 0){
-                Write-LogInfo "Unsupported kernel version: $currentKernelVersion"
-                return $global:ResultSkipped
-            }
+        $numberOfXDPQueues = Run-LinuxCmd -ip $receiverVMData.PublicIP -port $receiverVMData.SSHPort `
+                -username $user -password $password -command "ethtool -S $iFaceName  | grep xdp_drop | wc -l"
+        Write-LogInfo "Number of XDP DROP queues: $numberOfXDPQueues"
+        if ($numberOfXDPQueues -gt 0) {
+            Write-LogInfo "Kernel version supports XDP"
         } else {
-            Write-LogInfo "Unsupported distro: $($global:DetectedDistro)."
+            Write-LogWarn "Unsupported Kernel or unsupported distro $($global:DetectedDistro)"

```

Comment:
can we add the actual kernel info in the Warning message?  $global:initialkernelversion

Reply:
fixed.